trunk/NDHMS/template/HYDRO/hydro.namelist

```diff
@@ -147,6 +147,9 @@ channel_option = 3
 ! This option is currently only supported if using reach-based routing with UDMP=1.
 compound_channel = .FALSE.
 
+! Switch to activate channel-loss option (0=no, 1=yes) [Requires Kchan in RouteLink]
+! channel_loss_option = 0
```

Comment:
Another admittedly somewhat pedantic comment, but do we want the "_option" part on this?  Arguably clearer, but also inconsistent w/ existing "options."

I don't have a strong opinion, but figured I'd point it out now before there's no going back.

Reply:
My thinking here was to use `_option` to correspond to the existing `channel_option` parameter above, and also (as I chose an integer vs. a logical) to allow for alternate channel loss schemes that might be implemented in the future.

Reply:
Ah, got it.  Consistency w/in channel "options" is probably all we can hope for at this point anyway.   Sounds good and sorry to poke at that.